tests/cmdline/commands/test_code.py

```diff
@@ -236,6 +236,24 @@ def test_code_duplicate_ignore(run_cli_command, aiida_local_code_factory, non_in
     assert duplicate.description == ''
 
 
+def test_code_export(run_cli_command, aiida_local_code_factory, tmp_path, file_regression):
+    """Test export the code setup to str."""
+    code = aiida_local_code_factory('core.arithmetic.add', '/bin/cat', label='code')
```

Comment:
Can you please add some data for `append_text` and `prepend_text`. Ideally some multiline string with maybe some leading whitespace. So e.g.:
```
prepend_text = 'module load something\n    some command'
```

Reply:
Yes, added.